src/pluto/playground/components/source.cljs

```diff
@@ -45,7 +45,7 @@
                                            1000)))))))
       :reagent-render
       (fn [_]
-        [:div {:style {:height "55%"}}])})))
+        [:div {:style {:height "59%"}}])})))
```

Comment:
why not 60? :)

Reply:
Not sure why but with 60 there is a tiny scrollbar